monix-reactive/shared/src/test/scala/monix/reactive/internal/operators/OnErrorRestartWithBackoffSuite.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2014-2020 by The Monix Project Developers.
+ * See the project homepage at: https://monix.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package monix.reactive.internal.operators
+
+import monix.execution.exceptions.DummyException
+import monix.reactive.{BaseTestSuite, Observable}
+
+import scala.concurrent.duration._
+import scala.util.Success
+
+object OnErrorRestartWithBackoffSuite extends BaseTestSuite {
```

Comment:
Let's try to use constant backoff for shared tests - if it doesn't work then we'll settle on copy paste

Reply:
@Avasil I couldn't get the shared tests to work, even with a Constant Strategy, I think this is because the attempts also reset after a successful subscription.